pkg/ingester/ingester_v2.go

```diff
@@ -1493,6 +1511,11 @@ func (i *Ingester) getTSDBUsers() []string {
 	i.userStatesMtx.RLock()
 	defer i.userStatesMtx.RUnlock()
 
+	if i.TSDBState.closed {
```

Comment:
Please remove this check. No code using this method should run when ingester is Stopping.

Reply:
Good catch, I've removed it.